erpnext/accounts/doctype/sales_invoice/sales_invoice.js

```diff
@@ -486,7 +486,7 @@ erpnext.accounts.SalesInvoiceController = class SalesInvoiceController extends e
 	currency() {
 		var me = this;
 		super.currency();
-		if (this.frm.doc.timesheets) {
+		if (!this.frm.doc.is_return && this.frm.doc.timesheets) {
```

Comment:
The error here would better be solved by disabling the "No Copy" attribute of the _Timesheet Detail_ column of **Sales Invoice Timesheet**.

Reply:
Hi @barredterra,

I already had an idea 😅  and I set "no copy" then both problems were solved but I was a bit confused 🤔 but you are right. Now I simply add "no copy". Thank you very much for looking at and reviewing PR. 😄 

I will update the PR.